Editorial review: Document the Captured Surface Control API (#39982)
mdn/content: files/en-us/web/http/reference/headers/permissions-policy/index.md

- {{httpheader('Permissions-Policy/camera', 'camera')}} {{experimental_inline}}

>   - : Controls whether the current document is allowed to use video input devices. When this policy is disabled, the {{jsxref("Promise")}} returned by {{domxref("MediaDevices.getUserMedia", "getUserMedia()")}} will reject with a `NotAllowedError` {{DOMxRef("DOMException")}}.

Reviewer: ```suggestion
  - : Controls whether the current document is allowed to use video input devices.
    The {{jsxref("Promise")}} returned by {{domxref("MediaDevices.getUserMedia", "getUserMedia()")}} will reject with a `NotAllowedError` {{DOMxRef("DOMException")}} if the permission is not allowed.
```

- {{HTTPHeader('Permissions-Policy/captured-surface-control', 'captured-surface-control')}} {{experimental_inline}}

>   - : Controls whether or not the document is permitted to use the [Captured Surface Control API](/en-US/docs/Web/API/Screen_Capture_API/Captured_Surface_Control). When this policy is disabled, the promise returned by the API's main methods will reject with a `NotAllowedError` {{DOMxRef("DOMException")}} if permission is not obtained.

Reviewer: This sentence does not make sense.

> When this policy is disabled, the promise returned by the API's main methods will reject with a `NotAllowedError` {{DOMxRef("DOMException")}} if permission is not obtained.

It's not entirely your fault, the previous document also misuses disabled, and I may well have written that.

```suggestion
  - : Controls whether or not the document is permitted to use the [Captured Surface Control API](/en-US/docs/Web/API/Screen_Capture_API/Captured_Surface_Control).
    The promise returned by the API's main methods will reject with a `NotAllowedError` {{DOMxRef("DOMException")}} if permission is not granted.
```
